libraries/chain/controller.cpp

```diff
@@ -3607,7 +3615,7 @@ void controller_impl::on_activation<builtin_protocol_feature_t::replace_deferred
                ("name", itr->name)("adjust", itr->ram_correction)("current", current_ram_usage) );
       }
 
-      if (auto dm_logger = get_deep_mind_logger()) {
+      if (auto dm_logger = get_deep_mind_logger(false)) {
```

Comment:
We do not want to log this for dry-run transactions.

Reply:
This is called inside `controller_impl::on_activation<builtin_protocol_feature_t::                    replace_deferred>()`, which is for deferred transactions.